templates/vhost/_directories.erb

```diff
@@ -2,19 +2,17 @@
   <%- scope.setvar('_template_scope', {}) -%>
   ## Directories, there should at least be a declaration for <%= @docroot %>
   <%- @_directories.each do |directory| -%>
-    <%- if scope.function_versioncmp([@apache_version, '2.4']) >= 0 -%>
-      <%- if directory['allow'] and ! [ false, 'false', '' ].include?(directory['allow']) -%>
-        <%- scope.function_warning(["Apache::Vhost: Using allow is deprecated in your Apache version"]) -%>
-      <%- end -%>
-      <%- if directory['deny'] and ! [ false, 'false', '' ].include?(directory['deny']) -%>
-        <%- scope.function_warning(["Apache::Vhost: Using deny is deprecated in your Apache version"]) -%>
-      <%- end -%>
-      <%- if directory['order'] and ! [ false, 'false', '' ].include?(directory['order']) -%>
-        <%- scope.function_warning(["Apache::Vhost: Using order is deprecated in your Apache version"]) -%>
-      <%- end -%>
-      <%- if directory['satisfy'] and ! [ false, 'false', '' ].include?(directory['satisfy']) -%>
-        <%- scope.function_warning(["Apache::Vhost: Using satisfy is deprecated in your Apache version"]) -%>
-      <%- end -%>
+    <%- if directory['allow'] and ! [ false, 'false', '' ].include?(directory['allow']) -%>
+      <%- scope.function_warning(["Apache::Vhost: Using allow is deprecated in your Apache version"]) -%>
+    <%- end -%>
+    <%- if directory['deny'] and ! [ false, 'false', '' ].include?(directory['deny']) -%>
+      <%- scope.function_warning(["Apache::Vhost: Using deny is deprecated in your Apache version"]) -%>
+    <%- end -%>
+    <%- if directory['order'] and ! [ false, 'false', '' ].include?(directory['order']) -%>
+      <%- scope.function_warning(["Apache::Vhost: Using order is deprecated in your Apache version"]) -%>
+    <%- end -%>
+    <%- if directory['satisfy'] and ! [ false, 'false', '' ].include?(directory['satisfy']) -%>
+      <%- scope.function_warning(["Apache::Vhost: Using satisfy is deprecated in your Apache version"]) -%>
```

Comment:
In my PR I went further and simply killed the deprecated parameters. We're going for a major version bump: let's make it clean.